domain_model/Election.ts

```diff
@@ -39,6 +40,27 @@ export function electionValidation(obj:Election): string | null {
   return null;
 }
 
-export function removeHiddenFields(obj:Election):void {
+export function removeHiddenFields(obj:Election, electionRoll: ElectionRoll|null):void {
   obj.auth_key = undefined;
+  if (obj.state==='open' && electionRoll?.precinct){
```

Comment:
do we need to filter out precincts for every request?  Is this considered private information?  are we sure there aren't any routes that need this info?

Reply:
I think this might end up in a separate route just for ballot data. The intention was that when the voter requests the ballot to vote they only receive races that include their precinct, but only when the election is open so the other routes should still work. Another approach would be to include the list of approved races in the voter auth data and filter it client side.